### PR TITLE
refactor(ComickK): fix wrong website link and simplify code

### LINKED ISSUE #7151: Change the url for comick website
### Presence name

Comick

### Description

It's already in premid, it's just that the link for it needs to be updated
Here's the new link https://comick.app/

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by RisingSunLight42**
### Changelog

- Resolves #7151 
- Simplify a lot the codebase : remove lot of useless tests since there's some queries that are constants but were tested + simplify a switch case with only change to `presenceData.details`

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

comick_proof_1

comick_proof_2

comick_proof_3

comick_proof_4

comick_proof_5

</details>

### CODE REVIEW

File: websites/C/ComicK/presence.ts [hunk 1]
```diff
@@ -1,128 +1,102 @@
...
-				break;
-			}
-		}
+			break;
+		default:
+			if (Object.keys(staticPages).includes(arrPath[1]))
+				presenceData = { ...presenceData, ...staticPages[arrPath[1]] };
 	}
+
+	if (!image) presenceData.largeImageKey = "https://i.imgur.com/DeNqLCB.png";
```
[the remark this answers is not in the document]
Comment (RisingSunLight42) [reply websites/C/ComicK/presence.ts:99]: Yeah, true, I'll improve that

File: websites/C/ComicK/presence.ts [hunk 2]
```diff
@@ -1,128 +1,107 @@
...
-		}
+			break;
+		default:
+			if (Object.keys(staticPages).includes(arrPath[1]))
+				presenceData = { ...presenceData, ...staticPages[arrPath[1]] };
 	}
+
+	if (!image && presenceData.largeImageKey !== Assets.Logo)
+		presenceData.largeImageKey = Assets.Logo;
 	if (!buttons) delete presenceData.buttons;
```
Comment (DarkVillager) [inline websites/C/ComicK/presence.ts:105]: Same here, check if there are button (optional)

Comment (RisingSunLight42) [reply websites/C/ComicK/presence.ts:105]: I'll add it when I'll add strings

**Comment by RisingSunLight42** [comment]
Can't switch the branch to draft for now, but I'll add localized strings to the presence, should be done in few hours (or days if something happen that I can't predict).

**Review by DarkVillager — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**